libs/sdk-bindings/build.rs

```diff
@@ -1,6 +1,17 @@
 use anyhow::*;
+use camino::Utf8Path;
+use uniffi_kotlin_multiplatform::KotlinBindingGenerator;
 
 fn main() -> Result<()> {
-    uniffi_build::generate_scaffolding("./src/breez_sdk.udl").unwrap();
+    let udl_file = "./src/breez_sdk.udl";
+    let out_dir = Utf8Path::new("kmm");
+    uniffi_build::generate_scaffolding(udl_file).unwrap();
+    uniffi_bindgen::generate_external_bindings(
+        KotlinBindingGenerator {},
+        udl_file,
+        None::<&Utf8Path>,
```

Comment:
This seems to be the path for the uniff.toml file. Since it is passed with None I am guessing it uses some defaults.
Did you consider pointing to the uniffi.toml we have and add the kmp configuration? Then you can control the library name from there.

Reply:
Fixed in 52bbe0b0a75d115194d059543cf848030937fd3e